chartmogul.go

```diff
@@ -257,5 +257,6 @@ func (api API) req(req *gorequest.SuperAgent) *gorequest.SuperAgent {
 	}
 	return req.Timeout(timeout).
 		SetBasicAuth(api.ApiKey, "").
-		Set("Content-Type", "application/json")
+		Set("Content-Type", "application/json").
+		Set("User-Agent", "chartmogul-go/v3")
```

Comment:
Hmmm...could we get library version based on some constant? I see a situation already where we release new version and it's not updated over here.